src/com/xilinx/rapidwright/design/ModuleInst.java

```diff
@@ -236,23 +238,43 @@ public ArrayList<Site> getAllValidPlacements(){
 
 	/**
 	 * Places the module instance anchor at the newAnchorSite as well as all other 
-	 * instances and nets within the module instance at their relative offsets of the new site.
+	 * instances and nets within the module instance at their relative offsets of the new site. Note
+	 * that this method allows placement overlap by default.  See 
+	 * {@link #place(Site, boolean, boolean)} to disallow module overlap.
 	 * @param newAnchorSite The new site for the anchor of the module instance.
 	 * @return True if placement was successful, false otherwise.
 	 */
 	public boolean place(Site newAnchorSite){	
-		return place(newAnchorSite, false);
+	    return place(newAnchorSite, false);
 	}
-	
+
 	/**
 	 * Places the module instance on the module's anchor site (original location of the module).  
-	 * This is the same as place(getModule().getAnchor().getSite()). 
+	 * This is the same as place(getModule().getAnchor().getSite()). Note
+	 * that this method allows placement overlap by default.  See 
+	 * {@link #place(Site, boolean, boolean)} to disallow module overlap.
 	 * @return True if the placement was successful, false otherwise.
 	 */
 	public boolean placeOnOriginalAnchor() {
-		return place(module.getAnchor(), false);
+	    return place(module.getAnchor(), false);
```

Comment:
This calls `place()` with `allowOverlap=false` which is contrary to the javadoc which states that overlaps are allowed by default?

Reply:
Oh wait, never mind. That's for `skipIncompatible`!